docusaurus/docs-internals/editor-controls/multivalue-controls/motion.md

```diff
@@ -190,3 +253,132 @@ An array of animation names that should be disabled. In this case, you cannot se
   }
 }
 ```
+
+#### Default value examples
+
+In this example, the `motion` control that has the values for `"rotate"` and `"speed"` enabled by default and `"rotate"` is active.
+`"rotate"` has `"speed"` = `13` and `"direction"` = `"right"`, `"scale"` has `"speed"` = `7`.
+
+```js
+{
+  id: "motion", 
+  type: "motion",
+  default: {
+    active: "rotate",
+    rotateSpeed: 0.13,
+    rotateDirection: "right",
+    rotateEnabled: true,
+    scaleEnabled: true,
+    scaleSpeed: 7
+  }
+}
+```
+
+#### CSS examples
+
+Change the `.brz-text` element color with CSS using a `motion` control value.
+If `rotate` speed value >= 0.7, then we will colour `.brz-text` element to red, otherwise it will be blue.
+
+```js
+{
+  id: "motion",
+  type: "motion",
+  style: ({ value }) => {
+    if (value.rotate.speed >= 0.7) {
+      return {
+        "{{WRAPPER}} .brz-text": {
+          color: "red"
+        }
+      }
+    }
+
+    return {
+      "{{WRAPPER}} .brz-text": {
+        color: "blue"
+      }
+    }
+  }
+}
+```
+
+#### Usage in HTML example
+
+In the example below, we use the `motion` output value to determine when to render the label in the button element.
+We will render different label depending on `motion?.rotate?.speed`;
+
+```tsx
+import { Brizy } from "@brizy/core";
+import { Icon } from "./Icon";
+import React, { JSX } from "react";
+
+interface Props {
+  motion: {
+    active: undefined | "vertical" | "horizontal" | "transparency" | "blur" | "rotate" | "scale" | "mouseTrack" | "mouseTilt";
+    vertical: undefined | { speed: number; direction: "up" | "down"; viewport: { bottom: number; top: number; type: "viewport"; } };
+    horizontal: undefined | { speed: number; direction: "left" | "right"; viewport: { bottom: number; top: number; type: "viewport"; } };
+    transparency: undefined | { direction: "in" | "out" | "outIn" | "inOut"; level: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+    blur: undefined | { direction: "in" | "out" | "outIn" | "inOut"; level: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+    rotate: undefined | { direction: "left" | "right"; x: "left" | "center" | "right"; y: "top" | "center" | "bottom"; speed: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+    scale: undefined | { direction: "up" | "down" | "downUp" | "upDown"; speed: number; x: "left" | "center" | "right"; y: "top" | "center" | "bottom"; viewport: { bottom: number; top: number; type: "viewport"; } };
+    mouseTrack: undefined | { direction: "direct" | "opposite"; speed: number; };
+    mouseTilt: undefined | { direction: "direct" | "opposite"; speed: number; };
+  }
```

Comment:
review type

because we don't have first level `motion: {}`

our keys is prefix with control's `id`, as example: `active` is `motionActive`